plugin/pkg/scheduler/factory/factory.go

```diff
@@ -517,7 +517,11 @@ func (factory *ConfigFactory) MakeDefaultErrorFunc(backoff *util.PodBackoff, pod
 		if err == core.ErrNoNodesAvailable {
 			glog.V(4).Infof("Unable to schedule %v %v: no nodes are registered to the cluster; waiting", pod.Namespace, pod.Name)
 		} else {
-			glog.Errorf("Error scheduling %v %v: %v; retrying", pod.Namespace, pod.Name, err)
+			if _, ok := err.(*core.FitError); ok {
+				glog.V(4).Infof("Unable to schedule %v %v: no fit: %v; waiting", pod.Namespace, pod.Name, err)
```

Comment:
IMO the best balance of the concerns here is to do glog.Infof() but only log when the pod becomes pending initially, not every time the scheduler re-discovers that it is pending. This is more work but I think it's better than hiding this information behind V(4).